src/basic/selinux-util.c

```diff
@@ -199,11 +199,11 @@ int mac_selinux_get_create_label_from_exe(const char *exe, char **label) {
         if (!mac_selinux_use())
                 return -EOPNOTSUPP;
 
-        r = getcon(&mycon);
+        r = getcon_raw(&mycon);
         if (r < 0)
                 return -errno;
 
-        r = getfilecon(exe, &fcon);
+        r = getfilecon_raw(exe, &fcon);
```

Comment:
Hm, I also added a call to `mac_selinux_create_file_prepare` and found one more place that should be changed to `_raw`. Will push in a moment.